Cargo.toml

```diff
@@ -39,5 +39,8 @@ stdweb = { version = "0.4.9", optional = true }
 [target.wasm32-wasi.dependencies]
 libc = "0.2.54"
 
+[target.'cfg(any(target_env = "sgx", target_os = "uefi"))'.dependencies]
+lazy_static = { version = "1.3.0", features = ["spin_no_std"] }
```

Comment:
Note that `spin_no_std` feature depends on unstable language features, so SGX and UEFI targets will not work on stable compiler.

Reply:
Does it? I was able to `cargo build --target x86_64-fortanix-unknown-sgx` for getrandom without any issues on current stable Rust. And the `lazy_static` and `spin` crates don't seem to use unstable features.

Reply:
Hm, I was following `lazy_static` docs: https://docs.rs/lazy_static/1.3.0/lazy_static/#cargo-features Maybe it's out of date?

Reply:
Yup, was fixed in https://github.com/rust-lang-nursery/lazy-static.rs/pull/130 (for 1.2.0), but the docs didn't get updated until https://github.com/rust-lang-nursery/lazy-static.rs/pull/144 (after the current release).